packages/v4/patternfly-docs/content/accessibility/navigation/navigation.md

```diff
@@ -0,0 +1,77 @@
+
+
+
+## Testing
+
+At a minimum, a navigation component should meet the following criteria:
+
+<List isPlain>
+  <ListItem>
+    <Checkbox id="navigation-a11y-checkbox-1" label={<span>The navigation has an accessible name via the <code class="ws-code">aria-label</code></span>} description="This will differentiate each navigation when announced to users of assistive technologies when navigating through a page or a screen reader's rotor menu." />
+  </ListItem>
+  <ListItem>
+    <Checkbox id="navigation-a11y-checkbox-2" label="Standard keyboard navigation can be used to navigate between focusable elements within the navigation." description={<span><kbd>Tab</kbd> navigates to the next focusable element, and <kbd>Shift</kbd> + <kbd>Tab</kbd> navigates to the previous focusable element. Expandable items can be opened with <kbd>Enter</kbd> or <kbd>Space</kbd>. Flyout menu items use arrow keys to navigate between flyout items, and press <kbd>Escape</kbd> or left arrow to close a flyout. When the navigation is collapsed, you should not be able to navigate to any of the elements.</span>} />
+  </ListItem>
+  <ListItem>
+    <Checkbox id="navigation-a11y-checkbox-3" label={<span>For expandable or grouped navigation, an accessible name via the <code class="ws-code">title</code> or <code class="ws-code">aria-label</code> is used.</span>} description="This will differentiate each navigation when announced to users of assistive technologies when navigating through a page or a screen reader's rotor menu." />
+  </ListItem>
+</List>
+
+
+
+## React customization
+
+Various React props have been provided for more fine-tuned control over accessibility.
+
+| Prop | Applied to | Reason | 
+|---|---|---|
+| `aria-label` | `Nav` | 	Accessible label for the nav when there are multiple navs on the page |
+| `ariaLeftScroll` | `NavList` | Aria-label for the left scroll button |
+| `ariaRightScroll` | `NavList` | Aria-label for the right scroll button |
+| `title` | `NavExpandable` | Title shown for the expandable list |
+| `buttonProps` | `NavExpandable` | Additional props added to the NavExpandable `<button>`|
+| `srText` | `NavExpandable` | If defined, screen readers will read this text instead of the list title |
+
+
+### Expandable toggle props
+
+When a navigation has expandable content, `buttonProps` must be passed into the `NavExpandable` sub-component with an object as its value. This object should usually be passed in with a structure similar to the following:
+
+```noLive
+  buttonProps=({
+    id: ‘buttonId’, 
+    ‘aria-label’: 'Toggle label', 
+    ‘aria-labelledby’: 'titleId buttonId', 
+    ‘aria-expanded’: this.state.isExpanded
+  )}
+  ```
+
+
+## HTML/CSS customization
+
+| Attribute | Applied to | Outcome |
+| -- | -- | -- |
+| `aria-label="[landmark description]"` | `.pf-c-nav` |  Describes `<nav>` landmark. |
```

Comment:
FWIW I like having the information about what the value needs to be.

Reply:
I don't have particularly strong opinions about it, mostly just whatever we think makes the most sense and looks the cleanest. I don't mind taking it off for some that are more obvious, like labeling, and add anything missing to the description while keeping the content of some of them. For example, I don't think it makes sense to take off `aria-current="page"` because I don't think the average user would have any idea what to fill that with.

Reply:
If anything we could see if content design would have any recommendations.

@srambach to clarify, you prefer the syntax of `aria-label="[landmark description]"` for the first column right? Basically having a placeholder value for any attribute that should have something passed in (vs an attribute like `hidden`)?

Reply:
I like having a placeholder, yes.